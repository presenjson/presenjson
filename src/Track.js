import React from 'react';
import PropTypes from 'prop-types';

const Track = (props) => false;

export default Track;
