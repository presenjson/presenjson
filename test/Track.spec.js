import React from 'react';
import { shallow } from 'enzyme';
import Track from '../src/Track';

describe('Track', () => {
    it('renders nothing', () => {
        expect(
            shallow(
                <Track>
                    <div className="test" />
                </Track>
            )
        ).toMatchSnapshot();
    });
    it('sets default props', () => {
        expect(Track.defaultProps).toEqual({ solo: false });
    });
});
