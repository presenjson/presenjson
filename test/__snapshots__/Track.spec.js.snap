// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Track renders nothing 1`] = `""`;
